Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    - templateName: objc-string-h<br />      output: Localizable.h</pre><br />    - templateName: objc-string-m<br />      output: Localizable.m</pre> |
+| Language | Objective-C |
+| Author | Eric Slosser |
+
+## When to use it
+
+- When you need to generate *Objective-C* code.
+
+## Customization
+
+You can customize some elements of this template by overriding the following parameters when invoking `swiftgen`. See the [dedicated documentation](../../ConfigFile.md).
+
+| Parameter Name | Default Value | Description |
+| -------------- | ------------- | ----------- |
+| `noComments`   | N/A           | Setting this parameter will disable the comments describing the translation of a key. |
+| `headerName`   | "Localizable" | A generated .m file `#includes "{headerName}.h"` |
```

Comment:
That description of that parameter doesn't feel really clear to me.

1) Is that the name of the `.h` corresponding to the generated `.m`? If so, can't we assume it's gonna be the same name for the `.h` and  `.m` files? [EDIT] Oh wait, we don't have access to the name of the generated file from the context, that's why we have to provide it explicitly that way, right?
2) Wouldn't that description be clearer worded like "The name of the ObjC header generated by the `objc-string-h` template, that will be `#include`'d in the generated `.m` file — this parameter should have the same value as the `output` file name used for the `template: objc-string-h` entry in your config file"?

Reply:
I agree, I like your #2 better than mine, I'll put it in.